.github/actions/setup-rust/action.yaml

```diff
@@ -0,0 +1,36 @@
+name: 'Setup Rust env'
```

Comment:
I think we can remove this part as we don't build contracts on the wallet project

Reply:
✅